### PR TITLE
feat(auth): instead of text in the user agent field, now show an icon of a browser

### PR DESCRIPTION
**Pull request by serslon**
[FR-1873](https://frontegg.atlassian.net/browse/FR-1873) Sidekick/Audits - Change "user agent" to a logo

### CODE REVIEW

File: packages/audits/src/components/AuditsTableCell.tsx [hunk 1]
```diff
@@ -18,6 +20,29 @@ const AuditsTag: CellComponent = (props) => {
   );
 };
 
+// Logos take from the cdnjs site. The repository with the src is here https://github.com/alrra/browser-logos
+const UserAgent: CellComponent = ({ value }) => {
+  const browser = useMemo(() => {
+    const browser = new UAParser(value).getBrowser().name;
+    return browser && browsers.includes(browser.toLowerCase() ?? '') ? browser : 'Web';
```
Comment (frontegg-david) [inline packages/audits/src/components/AuditsTableCell.tsx:27]: - browser.toLowerCase() does not return undefined then the ?? '' will not work 
- what if the UAParser failed to parse the value? 
- I guess the getBrowser function return nullable value then the .name will throw exception

File: packages/audits/src/components/AuditsTableCell.tsx [hunk 2]
```diff
@@ -18,6 +20,29 @@ const AuditsTag: CellComponent = (props) => {
...
+  }, [value]);
+
+  return value ? (
+    <div className={classNames('fe-audits__severity')}>
```
Comment (frontegg-david) [inline packages/audits/src/components/AuditsTableCell.tsx:31]: why not to swap the return. do it like 

```
if(!value) {
  return null;
}

return <>
...
</>
```

File: packages/audits/src/components/AuditsTableCell.tsx [hunk 3]
```diff
@@ -18,6 +20,29 @@ const AuditsTag: CellComponent = (props) => {
...
+      <Popup
+        action='hover'
+        content={<div className={`${prefixCls}__useragent-hover`}>{value}</div>}
+        trigger={
+          <img
+            alt={browser}
+            src={`//cdnjs.cloudflare.com/ajax/libs/browser-logos/69.0.4/${browser.toLowerCase()}/${browser.toLowerCase()}_${sizeOfIcon}x${sizeOfIcon}.png`}
```
Comment (frontegg-david) [inline packages/audits/src/components/AuditsTableCell.tsx:38]: calculate the url outside the JSX code

**Review by frontegg-david — CHANGES_REQUESTED** [review]

File: packages/audits/src/components/AuditsTableCell.tsx (hunk 3 above)
Comment (serslon) [reply packages/audits/src/components/AuditsTableCell.tsx:38]: I don't see the reason to do it. But I'll do it.

File: packages/audits/src/components/AuditsTableCell.tsx (hunk 1 above)
Comment (serslon) [reply packages/audits/src/components/AuditsTableCell.tsx:27]: Yes, the getBrowser function returns a nullable value. I'll fix it.

File: packages/audits/src/components/AuditsTableCell.tsx (hunk 3 above)
Comment (frontegg-david) [reply packages/audits/src/components/AuditsTableCell.tsx]: It's just long lines in JSX code

**Review by frontegg-david — APPROVED** [review]

**Pull request merged**